frontend/src/app/core/setup/globals/onboarding/onboarding_tour.ts

```diff
@@ -121,7 +115,7 @@ export function start(name:OnboardingTourNames, project?:ProjectName):void {
       startTour(scrumTaskBoardTourSteps());
       break;
     case 'homescreen':
-      initializeTour('startProjectTour', '.widget-box--blocks--buttons a', true);
+      initializeTour('startProjectTour', '.widget-box--blocks--buttons a');
```

Comment:
I think we can cleanup a bit more here: The second paramter (`.widget-box--blocks--buttons a`)  is not necessary any more and since it was the only place that used the second paramter, it can probably also be removed from the method definition